### PR TITLE
fingerprint fix for sending app to background

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -88,7 +88,7 @@ PODS:
     - GoogleUtilities/Logger
```
Comment (KedziaPawel) [inline ios/Podfile.lock:88]: why it is downgrading all libs?

File: src/navigators/Navigator.tsx [hunk 2]
```diff
@@ -93,7 +94,7 @@ class Navigator extends React.Component<Props, State> {
     if (__DEV__) {
       return false;
     }
-    return !isAuthenticated && isTxPasswordSet && isPinSet;
+    return !isAuthenticated && isTxPasswordSet && isPinSet && AppState.currentState === 'active';
```
Comment (KedziaPawel) [inline src/navigators/Navigator.tsx:97]: does it allow the user to hack pin by clicking quickly on the tab bar?

Comment (Kamill90) [reply src/navigators/Navigator.tsx:97]: could not reproduce

File: ios/Podfile.lock (hunk 1 above)
Comment (Kamill90) [reply ios/Podfile.lock:88]: I can't say actually. I needed to reinstall Pods recently. That's what I got

Comment (KedziaPawel) [reply ios/Podfile.lock:88]: what pods version do you have?

Comment (Kamill90) [reply ios/Podfile.lock:88]: 1.9.3

Comment (KedziaPawel) [reply ios/Podfile.lock:88]: strange, me too

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**